src/components/Video/Video.js

```diff
@@ -141,9 +152,16 @@ export default class Video extends React.Component {
         </g>
       </svg>
     );
+    const videoWrapperClassName = classnames('ibm--video-wrapper', {
+      ' active': this.state.isVideoWrapperActive,
+    });
 
     return (
-      <div className="ibm--video-wrapper" onClick={this.onClick}>
+      <div
+        className={videoWrapperClassName}
+        onClick={this.handlePlaySecondVideo}
+        onKeyDown={this.handlePlaySecondVideo}
+        tabindex="0">
```

Comment:
```suggestion
        tabIndex="0">
```

Also need to replace `class` with `className` in some SVG elements - Just found them looking at the console.